entity/src/main/java/com/wilma/entity/users/Resume.java

```diff
@@ -0,0 +1,53 @@
+package com.wilma.entity.users;
+
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.Date;
+
+
+@Getter
+@Setter
+@AllArgsConstructor
+@Entity
+@Table(name = "files")
```

Comment:
I feel this should match the class name
If it's for many files maybe we should name the class 'File', or have a superclass called 'File' the 'Resume' extends from